includes/class-sensei-frontend.php

```diff
@@ -1732,7 +1732,7 @@ public function sensei_process_registration(){
 		}
 
 		// notify the user
-		wp_new_user_notification( $user_id, $new_user_password );
+		wp_new_user_notification( $user_id, null, 'user' );
```

Comment:
Thanks for the feedback! 

I see in your [commit](https://github.com/Automattic/sensei/pull/1844/commits/090efad2e79fd47da6eb35ca095544be1b67ca4a) that addresses this, you use 'admin' instead of 'user' for the last parameter.

I read the [documentation for wp_new_user_notification](https://developer.wordpress.org/reference/functions/wp_new_user_notification/) a little closer.

Is there a reason not to have an email sent to the user? For example, why not have it sent to both the admin and the user using 'both' as the parameter? 

Perhaps I'm not fully understanding the purpose of this function.